server/utilities/userInput.js

```diff
@@ -34,6 +34,18 @@ const validatePassword = (password) => {
   return passwordPattern.test(password);
 };
 
+/**
```

Comment:
This function doesn't actually confirm that the file is an image. I thought we discussed this?

Reply:
Yes, we did.

Reply:
I worked with Victor on that. He suggested that we can implement something on the frontend to do the confirmation.

Reply:
I believe cloudinary checks for that. So I have implemented a fail-safe for that using _try... catch_ method. I believe the edge case has been covered.

Reply:
<img width="1031" alt="screen shot 2018-11-02 at 4 41 09 pm" src="https://user-images.githubusercontent.com/31534129/47925310-26beef80-debe-11e8-8eb4-7241ee9eccbe.png">

Reply:
Ah, yeah you're right. This should be cloudinary's problem